src/schema/v2/filterArtworksConnection.ts

```diff
@@ -324,7 +324,13 @@ const filterArtworksConnectionType = connectionDefinitions({
       type: new GraphQLNonNull(GraphQLID),
       description: "The ID of the object.",
       resolve: ({ gravityOptions }) => {
-        const { offset, ...filterOptions } = gravityOptions
+        const {
+          aggregations,
```

Comment:
Discarding pagination and aggregation info, we're left with just filter options. This gets subsequent paging to have the same `id` as the first one.